src/main/java/nextstep/member/ui/MemberController.java

```diff
@@ -43,7 +43,7 @@ public ResponseEntity<MemberResponse> deleteMember(@PathVariable Long id) {
     }
 
     @GetMapping("/members/me")
-    public ResponseEntity<MemberResponse> findMemberOfMine(@AuthenticationPrincipal LoginMember loginMember) {
+    public ResponseEntity<MemberResponse> findMemberOfMine(@ AuthenticationPrincipal LoginMember loginMember) {
```

Comment:
어떤 이유로 공백이 추가되었을 까요? 👀 
```suggestion
    public ResponseEntity<MemberResponse> findMemberOfMine(@AuthenticationPrincipal LoginMember loginMember) {
```